appeals/e2e/.env.sample

```diff
@@ -1,7 +1,9 @@
 BASE_URL=
+API_BASE_URL=
```

Comment:
Required so that Cypress can call the API in order to create cases and make other direct changes to the data